git_machete/docs.py

```diff
@@ -154,13 +175,9 @@
         No branch will be deleted unless explicitly confirmed by the user (or unless `-y/--yes` option is passed).
         Equivalent of `git machete github sync` if invoked with `-H` or `--checkout-my-github-prs`.
 
-        To allow GitHub API access for private repositories (and also to perform side-effecting actions like opening a PR, even in case of public repositories),
-        a GitHub API token with `repo` scope is required, see https://github.com/settings/tokens. This will be resolved from the first of:
+{textwrap.indent(github_api_access, "        ")}
```

Comment:
Whoops sth broken here:

![image](https://user-images.githubusercontent.com/3383210/168443673-361454a6-2846-47b0-b9e0-aa9cfbfe9439.png)

Reply:
(as mentioned previously, this `<b>` is redundant here anyway IMO)

Reply:
Probably will get fixed once `<b>` tag above is sorted out... will check